weather_mv/loader_pipeline/ee.py

```diff
@@ -486,21 +487,46 @@ def convert_to_asset(self, queue: Queue, uri: str):
                 channel_names = []
                 file_name = f'{asset_name}.csv'
 
-                df = xr.Dataset.to_dataframe(ds)
-                df = df.reset_index()
-                # NULL and NaN create data-type mismatch issue in ee therefore replacing all of them.
-                # fillna fills in NaNs, NULLs, and NaTs but we have to exclude NaTs.
-                non_nat = df.select_dtypes(exclude=['datetime', 'timedelta', 'datetimetz'])
-                df[non_nat.columns] = non_nat.fillna(-9999)
+                shape = math.prod(list(ds.dims.values()))
```

Comment:
Check if Xarray has a built-in method for this. I think there is one on DataArrays.

Reply:
We can use `ds.prod` or `DataArray.prod` if we convert Frozen to ds or da. I think direct multiplication is simple instead.

Reply:
Ok, SGTM.